.circleci/organise_doc.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const fs = require('fs')
+const args = require('minimist')(process.argv.slice(2))
+
+// get the output directory
+const typedocJSON = require('../typedoc.json')
+const typedocOutputDir = typedocJSON.out
+
+// root dir is the absolute path to use, provided by CI
+const rootDir = args.rootdir
+if (rootDir === '' || rootDir == null){
+  console.error('rootDir argument must be defined');
+  process.exit(1);
+}
+
+// source directories where typedoc put the doc
+const sourceDir = `${rootDir}/${typedocOutputDir}`
+
+// the tag being built and empty or null if none
+const tag = args.tag
+
+// if no tag, doc will go in latest directory by default
+const targetDir = `${rootDir}/docs/${(tag === '' || tag == null) ? 'latest' : tag}`
```

Comment:
this type of lines can be replaces by `!var` as empty string, null and undefined evaluate to false. For example: `(tag === '' || tag == null)` becomes `!tag`